storb/miner/miner.py

```diff
@@ -197,3 +185,148 @@ async def iter_response():
         return StreamingResponse(
             iter_response(), media_type=f"multipart/mixed; boundary={boundary}"
         )
+
+    async def ack_challenge(self, request: protocol.NewChallenge):
+        """Acknowledges a challenge from a validator,
+        verifies it, and enqueues it upon success.
+        """
+        logger.info(f"Received challenge {request.challenge_id}")
+
+        logger.info(
+            f"Verifying challenge {request.challenge_id} wih validator {request.validator_id} and signature {request.signature}"
+        )
+
+        logger.debug(f"challenge: {request}")
+        try:
+            deadline_dt = datetime.datetime.fromisoformat(request.challenge_deadline)
+        except ValueError:
+            logger.error("Invalid challenge_deadline format. Must be valid ISO8601.")
```

Comment:
```suggestion
            logger.error("Invalid challenge_deadline format. Must be valid ISO 8601.")
```

Must be valid ISO 8601 what? String? Make this error message a bit more clear